deployment/requirements/dev.txt

```diff
@@ -84,6 +84,8 @@ gitpython==3.1.26 \
 iniconfig==1.1.1 \
     --hash=sha256:011e24c64b7f47f6ebd835bb12a743f2fbe9a26d4cecaa7f53bc4f35ee9da8b3 \
     --hash=sha256:bc3af051d7d14b2ee5ef9969666def0cd1a000e121eaea580d4a313df4b37f32
+inject==4.3.1 \
```

Comment:
Yes, we want that to be in repo - that way exactly know which packages are used. It's similar mechanism to: https://stackoverflow.com/questions/44297803/what-is-the-role-of-the-package-lock-json

Another benefit of doing things that way is that we have one-command-requrements-version update mechanism. 
If you want to pin specific package so that it won't get updated, the `*.in` files are also capable of it:
```
test-app-3 ; python_version > '3'
linux-app ; sys_platform == 'linux'
requests~=2.12.5  # https://github.com/shazow/urllib3/issues/1104
```
More on that [here](https://code.kiwi.com/our-comprehensive-guide-to-python-dependencies-8a5a4366a563#2979) which also is linked in readme.